prov/efa/src/rxr/rxr_ep.c

```diff
@@ -728,6 +728,25 @@ void rxr_ep_wait_send(struct rxr_ep *rxr_ep)
 	ofi_mutex_unlock(&rxr_ep->util_ep.lock);
 }
 
+/**
+ * @brief Destroy the ibv_cq_ex in rxr_ep
+ * 	Will be removed when rxr_ep and efa_ep are fully separated
+ *
+ * @param[in] rxr_ep Pointer to the rxr_ep.
+ * @param[in] ibv_cq_ex Pointer to ibv_cq_ex.
+ */
+static int rxr_ep_release_device_cq(struct rxr_ep *rxr_ep)
+{
+	struct efa_ep *efa_ep = container_of(rxr_ep->rdm_ep, struct efa_ep, util_ep.ep_fid);
+
+	if (efa_ep->scq)
```

Comment:
I think we always allocate `efa_ep->scq`. Right?, so this should be an assertion.